installer/templates/new/jobs/text.exs

```diff
@@ -3,5 +3,10 @@ use Kitto.Job.DSL
 job :text, every: {4, :seconds} do
   phrases = ["This is your shiny new dashboard", "Built on the Kitto Framework"]
 
-  broadcast! :text, %{text: (phrases |> Enum.shuffle |> Enum.take(1))}
+  broadcast! :text, 
+    %{
+      text: (phrases |> Enum.shuffle |> Enum.take(1)),
+      moreinfo: "This is where you can put dynamic text",
+      specialStyle: "widget-text-custom"
```

Comment:
We should leave the job as is. We don't have to update every field of the widget as long as it's documented at https://github.com/kittoframework/kitto/wiki/Available-Widgets#text